lib/tools/src/cover.erl

```diff
@@ -1143,7 +1143,10 @@ call(Request) ->
     Ref = erlang:monitor(process,?SERVER),
     receive {'DOWN', Ref, _Type, _Object, noproc} -> 
 	    erlang:demonitor(Ref),
-            {ok,_} = start(),
+            case start() of
+                {ok,_} -> ok;
+                {already_started,_} -> ok
```

Comment:
I didn't notice this problem; Dialyzer found it:

```suggestion
                {error,{already_started,_}} -> ok
```

Reply:
Hmm, then my testing was incorrect because I was unable to reproduce the problem with the original patch…